<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>directory</class>
 <widget class="QMainWindow" name="directory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>646</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QListWidget" name="myListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QMenu" name="menuCreate">
     <property name="title">
      <string>Create</string>
     </property>
     <addaction name="actionNew_Document"/>
     <addaction name="actionNew_Folder"/>
    </widget>
    <addaction name="menuCreate"/>
   </widget>
   <widget class="QMenu" name="menuGo_To">
    <property name="title">
     <string>Go To</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionUri"/>
   </widget>
   <addaction name="menuOperations"/>
   <addaction name="menuGo_To"/>
  </widget>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNew_Document">
   <property name="text">
    <string>New Document</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionUri">
   <property name="text">
    <string>Uri</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
